django_lti_tool_provider/tests/test_views.py

```diff
@@ -1,5 +1,6 @@
 import ddt
 from django.contrib.auth import login, authenticate
+from django.utils.importlib import import_module
```

Comment:
Is this any different than import_module() in the standard library?  I'd personally find it more natural to import it from importlib directly.